doc/man1/rehash.pod

```diff
@@ -72,8 +72,8 @@ B<OPENSSL> environment variable to the full pathname.
 Any program can be used, it will be invoked as follows for either
 a certificate or CRL:
 
-  $OPENSSL x509 -hash -fingerprint -noout -in FILENAME
-  $OPENSSL crl -hash -fingerprint -noout -in FILENAME
+  $ openssl x509 -hash -fingerprint -noout -in FILENAME
+  $ openssl crl -hash -fingerprint -noout -in FILENAME
```

Comment:
This isn't using "$" for a shell command example. It is describing how the c_rehash script will invoke external programs based on the contents of the "$OPENSSL" environment variable.